pkg/relayer/proxy/errors.go

```diff
@@ -5,19 +5,13 @@ import (
 )
 
 var (
-	codespace                                     = "relayer_proxy"
-	ErrRelayerProxyUnsupportedRPCType             = sdkerrors.Register(codespace, 1, "unsupported rpc type")
-	ErrRelayerProxyInvalidSession                 = sdkerrors.Register(codespace, 2, "invalid session in relayer request")
-	ErrRelayerProxyInvalidSupplier                = sdkerrors.Register(codespace, 3, "supplier does not belong to session")
-	ErrRelayerProxyUndefinedSigningKeyNames       = sdkerrors.Register(codespace, 4, "supplier signing key names are undefined")
-	ErrRelayerServicesConfigsUndefined            = sdkerrors.Register(codespace, 5, "services configurations are undefined")
-	ErrRelayerProxyInvalidRelayRequest            = sdkerrors.Register(codespace, 6, "invalid relay request")
-	ErrRelayerProxyInvalidRelayResponse           = sdkerrors.Register(codespace, 7, "invalid relay response")
-	ErrRelayerProxyServiceEndpointNotHandled      = sdkerrors.Register(codespace, 8, "service endpoint not handled by relayer proxy")
-	ErrRelayerProxyUnsupportedTransportType       = sdkerrors.Register(codespace, 9, "unsupported proxy transport type")
-	ErrRelayerProxyInternalError                  = sdkerrors.Register(codespace, 10, "internal error")
-	ErrRelayerProxyMissingSupplierOperatorAddress = sdkerrors.Register(codespace, 11, "supplier operator address is missing")
-	ErrRelayerProxyUnknownSession                 = sdkerrors.Register(codespace, 12, "relayer proxy encountered unknown session")
-	ErrRelayerProxyRateLimited                    = sdkerrors.Register(codespace, 13, "offchain rate limit hit by relayer proxy")
-	ErrRelayerProxyUnclaimRelayPrice              = sdkerrors.Register(codespace, 14, "failed to unclaim relay price")
+	codespace                                = "relayer_proxy"
+	ErrRelayerProxyInvalidSession            = sdkerrors.Register(codespace, 1, "invalid session in relayer request")
+	ErrRelayerServicesConfigsUndefined       = sdkerrors.Register(codespace, 2, "services configurations are undefined")
+	ErrRelayerProxyServiceEndpointNotHandled = sdkerrors.Register(codespace, 3, "service endpoint not handled by relayer proxy")
+	ErrRelayerProxyUnsupportedTransportType  = sdkerrors.Register(codespace, 4, "unsupported proxy transport type")
+	ErrRelayerProxyInternalError             = sdkerrors.Register(codespace, 5, "internal error")
+	ErrRelayerProxyUnknownSession            = sdkerrors.Register(codespace, 6, "relayer proxy encountered unknown session")
+	ErrRelayerProxyRateLimited               = sdkerrors.Register(codespace, 7, "offchain rate limit hit by relayer proxy")
+	ErrRelayerProxyUnclaimRelayPrice         = sdkerrors.Register(codespace, 8, "failed to unclaim relay price")
```

Comment:
I'm not 100% sure what "Unclaim relay price" is by just reading it.

As someone acquainted with the protocol, I think we should both update the variable name and the text message.